synfig-studio/plugins/lottie-exporter/lottie-exporter.py

```diff
@@ -20,6 +20,53 @@
 import settings
 import argparse
 
+def calc_font_data(lottie,layer):
+    """
+    Calculates the font data used in the outermost canvas
+
+    Args:
+        lottie (dict)       : Outermost dictionary used in Lottie file
+        layer  (common.Layer.Layer) : Synfig format text layer
+
+    Returns:
+        (None)
+    """
+    default_list = {
+        "origin": 0,
+        "fPath": "",
+        "fClass": "",
+        "fFamily": "",
+        "fWeight": "",
+        "fName": "",
+        "ascent": 75.9994506835938
```

Comment:
Can you add a comment for this "magic" number?

Reply:
There is no available documentation for the font description dictionary, rest of the elements were obvious but I wasn't able to figure out the purpose of "ascent".